static/app/views/dashboards/widgetCard/spansWidgetQueries.tsx

```diff
@@ -170,7 +170,7 @@ function SpansWidgetQueriesProgressiveLoadingImpl({
               // the best effort query has completed.
               children({
                 ...(bestEffortChildrenProps ?? preflightProps),
-                isProgressivelyLoading: preflightProps.loading,
```

Comment:
I'm not actually using `isProgressivelyLoading` at the moment, but `loading` is being used. I'm removing it from here but I'll revisit it when I come back to fixing up the UI from the higher component.